### PR TITLE
Set auto assign by github bot

### PR DESCRIPTION
**Pull request by x86chi**
Screen Shot 2020-08-28 at 3 30 07 PM

You can see preview at https://github.com/planetarium/auto-assign-test/pull/2

### CODE REVIEW

**Review by longfin — APPROVED** [review]

**Comment by longfin** [comment]
@x86chi I guess you need to add `[skip changelog]` to your commit message to bypass changelog checking.

**Comment by dahlia** [comment]
Does this bot work for draft pull requests as well?

**Comment by x86chi** [comment]
> Does this bot work for draft pull requests as well?

It not works draft PR.

**Review by longfin — APPROVED** [review]

File: .github/auto_assign.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+addReviewers: true
+addAssignees: author
+
+reviewers:
+  - dahlia
+  - longfin
+  - limebell
+  - earlbread
+  - riemannulus
+  - moreal
```
Comment (dahlia) [inline .github/auto_assign.yml:10]: Would the syntax for teams work?

```suggestion
- planetarium/libplanet
```

Comment (riemannulus) [reply .github/auto_assign.yml:10]: Doesn't this disappear all of the reviewer signing when one person reviews it?

Comment (x86chi) [reply .github/auto_assign.yml:10]: Yes. I hear to @moreal.

I think insert `planetarium/libplanet` to newline for Visibility

Screen Shot 2020-08-28 at 3 38 58 PM

**Review by limebell — APPROVED** [review]

**Comment by dahlia** [comment]
@x86chi Could you rebase this on the up-to-date main branch?

**Comment by x86chi** [comment]
@dahlia I did rebase.

**Review by dahlia — DISMISSED** [review]

**Review by longfin — APPROVED** [review]

**Review by limebell — APPROVED** [review]

**Review by earlbread — DISMISSED** [review]

**Review by longfin — APPROVED** [review]

**Review by dahlia — APPROVED** [review]

**Pull request merged**